frodo.js

```diff
@@ -1,17 +1,16 @@
 /**
```

Comment:
SCRIPT-Nice job on the comprehensive test coverage, it gives a lot of confidence in the stability of the new feature.